src/test/java/ro/msg/learning/shop/CreateOrderIntegrationTests.java

```diff
@@ -0,0 +1,121 @@
+package ro.msg.learning.shop;
```

Comment:
The convention in testing is that the package name should be the same as the equivalent from the class you are testing